src/common/options.cc

```diff
@@ -599,7 +599,7 @@ std::vector<Option> get_global_options() {
     .set_description(""),
 
     Option("ms_tcp_prefetch_max_size", Option::TYPE_INT, Option::LEVEL_ADVANCED)
-    .set_default(4096)
+    .set_default(4_K)
```

Comment:
This is excellent. Where are these literals defined?

Reply:
right in this file. line:144.